ArduPlane/Parameters.h

```diff
@@ -93,14 +93,14 @@ class Parameters {
         k_param_scheduler,
         k_param_relay,
         k_param_takeoff_throttle_delay,
-        k_param_skip_gyro_cal, // unused
+        k_param_takeoff_rudder_mix,
```

Comment:
I just thought that since they are not used and been there for ages, like years. They are serving no purpose not even the one they were there in the first place for, i.e smooth parameter version update. Is there any plan on doing something with these indices I am not aware of?